dlls/player.cpp

```diff
@@ -117,6 +117,7 @@ TYPEDESCRIPTION	CBasePlayer::m_playerSaveData[] =
 	DEFINE_FIELD( CBasePlayer, m_pTank, FIELD_EHANDLE ),
 	DEFINE_FIELD( CBasePlayer, m_iHideHUD, FIELD_INTEGER ),
 	DEFINE_FIELD( CBasePlayer, m_iFOV, FIELD_INTEGER ),
+	DEFINE_FIELD( CBasePlayer, m_flNextAttack, FIELD_TIME ),
```

Comment:
Won't this also fix quickgauss without any changes in the quickgauss commit?

Reply:
yea that's what happens when you fix quickgauss first, then other shit sry :))

Reply:
i'd still keep it since it's inline with the older HLSDK this way (the fix quickgauss commit)

https://github.com/a1batross/HLSDK_ancient/blob/master/DLLS/GAUSS.CPP

Reply:
Nah you baited me. xd Fixed quickgauss commit has to be there.

See matherunner's commentary here, that I've used as a base for the fix a year ago but already forgot: https://www.jwchong.com/hl/weapons.html#quickgauss

```
Notice how the highlighted line is inexplicably commented out, so that m_flStartCharge does not get saved to the savestate. When the game is loaded from a save, this field will be zero. The calculation of current charging time is done by gpGlobals->time - m_pPlayer->m_flStartCharge throughout dlls/gauss.cpp, with gpGlobals->time being much larger than 4 in most cases.
```

That's why in older HLSDK it's not present, ye. D